app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -167,10 +168,14 @@ ImportModelClass to_import_model(const std::string& g4_model_name)
 /*!
  * Safely switch from \c G4PhysicsVectorType to \c ImportPhysicsVectorType .
  * [See G4PhysicsVectorType.hh]
+ *
+ * Geant4 v11 has a different set of G4PhysicsVectorType enums.
  */
 ImportPhysicsVectorType
 to_import_physics_vector_type(G4PhysicsVectorType g4_vector_type)
 {
+#if defined(G4VERSION_NUMBER) && G4VERSION_NUMBER < 1100
```

Comment:
`G4VERSION_NUMBER` should always be defined and available through that header. I've added the safeguard just in case, so it is easier to spot a problem. 

Regarding our include (which I think leaving it inside `app/geant-exporter` is ok since that is the only place we will check that), do you have any naming preferences? `CeleritasG4Version.hh`?